src/goto-symex/symex_goto.cpp

```diff
@@ -359,26 +360,37 @@ void goto_symext::phi_function(
   const statet::goto_statet &goto_state,
   statet &dest_state)
 {
-  // go over all variables to see what changed
-  std::unordered_set<ssa_exprt, irep_hash> variables;
+  auto ssa_of_current_name =
+    [&](const std::pair<irep_idt, std::pair<ssa_exprt, unsigned>> &pair) {
+      return pair.second.first;
+    };
+
+  auto dest_state_names_range =
+    make_range(dest_state.level2.current_names)
+      .filter(
+        [&](const std::pair<irep_idt, std::pair<ssa_exprt, unsigned>> &pair) {
```

Comment:
⛏ I'd consider `const statet::value_type &` here. 🌊

Reply:
That's actually `statet::renaming_levelt::current_namest::value_type` so not really better I think